src/clusterfuzz/_internal/tests/core/base/tasks_test.py

```diff
@@ -46,33 +46,23 @@ def test_initialize_trusted_task(self):
 
   def test_initialize_untrusted_task(self):
     """Tests that an untrusted task is initialized properly."""
-    self_link = f'https://www.googleapis.com/storage/v1/b/{self.bucket}/o/{self.path}'
+    self_link = f'https://www.googleapis.com/storage/v1/b/{self.bucket}23/o/{self.path}23'
     self.message.attributes = {
         'kind': 'storage#object',
+        'bucket': self.bucket,
+        'name': self.path,
         'selfLink': self_link,
         'command': self.command,
         'argument': self.argument,
         'job': self.job,
     }
     task = tasks.initialize_task([self.message])
     self.assertFalse(isinstance(task, tasks.PubSubTask))
-    self.assertEqual(task.command, 'uworker_postprocess')
-    self.assertEqual(task.argument, '/mybucket/worker.output')
+    self.assertEqual(task.command, 'postprocess')
+    # !!! Is this bad for local development.
```

Comment:
@oliverchang Is it a problem if the task is using gs://bucket/data instead of /bucket/data ?
I don't think this will be executed in local development since pub/sub notifications for GCS can't really be done locally.